google/cloud/security/inventory/inventory_loader.py

```diff
@@ -18,7 +18,7 @@
 
 Usage:
   $ forseti_inventory \\
-      --inventory_groups <true|false> (optional) \\
+      --pipeline_config (required) \\
       --groups_service_account_key_file <path to file> (optional)\\
```

Comment:
Does it make sense to put the groups account keyfile/super admin email in the pipeline config, like in a "metadata" property?

e.g.

```
- resource: groups
   should_run: false
   service_acct_keyfile: path/to/keyfile
   domain_superadmin_email: superadmin@domain.local
```